tokio-signal/examples/multiple.rs

```diff
@@ -0,0 +1,38 @@
+//! A small example of how to listen for two signals at the same time
```

Comment:
This should be guarded with `#![cfg(unix)]`. I think the Windows CI is failing because we test examples, but we didn't used to build them in the old repo